frontend/shared-components/submit-button/submit-button.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import styles from './submit-button.module.css'
+
+interface SubmitButtonProps { text?: string }
+export default function SubmitButton({ text }: SubmitButtonProps) {
+  const displayText: string = text || 'Submit'
+
+  return (
+    <input 
```

Comment:
lets keep this as a simple button with an onclick using form and input submit has other side effects that can cause lots of issues down the road, better to simplify and just use a button.

Reply:
Oh okay, sounds good! Will do